dist/lucca-ui.js

```diff
@@ -872,6 +873,437 @@
+	"class=\"lui regular nguibs-ui-select\" on-select=\"addSelectedUser()\" on-remove=\"onRemove()\" ng-disabled=\"controlDisabled\">" +
+	"<ui-select-match placeholder=\"VAR_TRAD Sélectionner un utilisateur...\">{{$item.firstName}} {{$item.lastName}} " +
+	"<span ng-if=\"$item.hasHomonyms\" ng-repeat=\"property in properties\">&lt{{getProperty($item, property)}}&gt</span>" +
+	"<span ng-if=\"$item.isFormerEmployee\">&lt;VAR_TRAD Parti(e) le {{$item.dtContractEnd | luifMoment: 'll'}}&gt;</span>" +
+	"</ui-select-match>" +
+	uiSelectChoicesTemplate +
+	"</ui-select>";
+
+
+	angular.module('lui.directives')
+	.directive('luidUserPicker', function () {
+		return {
+			restrict: 'E',
+			controller: "luidUserPickerController",
+			template: userPickerTemplate,
+			require: "luidUserPicker",
+			scope: {
+				/*** STANDARD ***/
+				onSelect: "&",
+				onRemove: "&",
+				controlDisabled: "=",
+				/*** FORMER EMPLOYEES ***/
+				showFormerEmployees: "=", // boolean
+				/*** HOMONYMS ***/
+				homonymsProperties: "@", // list of properties to handle homonyms
+				/*** CUSTOM FILTER ***/
+				customFilter: "&", // should be a function with this signature: function(user){ return boolean; } 
+				/*** OPERATION SCOPE ***/
+				appId: "@",
+				operation: "@"
+			},
+			link: function (scope, elt, attrs, ctrl) {
+				if (attrs.homonymsProperties) {
+					scope.properties = attrs.homonymsProperties.split(',');
+				}
+				else {
+					scope.properties = DEFAULT_HOMONYMS_PROPERTIES;
+				}
+				ctrl.isMultipleSelect = false;
+				ctrl.asyncPagination = false;
+				ctrl.useCustomFilter = !!attrs.customFilter;
+			}
+		};
+	})
+
+	.directive('luidUserPickerMultiple', function () {
+		return {
+			restrict: 'E',
+			controller: "luidUserPickerController",
+			template: userPickerMultipleTemplate,
+			require: "luidUserPickerController",
+			scope: {
+				/*** STANDARD ***/
+				onSelect: "&",
+				onRemove: "&",
+				controlDisabled: "=",
+				/*** FORMER EMPLOYEES ***/
+				showFormerEmployees: "=", // boolean
+				/*** HOMONYMS ***/
+				homonymsProperties: "@", // list of properties to handle homonyms
+				/*** CUSTOM FILTER ***/
+				customFilter: "&", // should be a function with this signature: function(user){ return boolean; } 
+				/*** OPERATION SCOPE ***/
+				appId: "@",
+				operation: "@"
+			},
+			link: function (scope, elt, attrs, ctrl) {
+				if (attrs.homonymsProperties) {
+					scope.properties = attrs.homonymsProperties.split(',');
+				}
+				else {
+					scope.properties = DEFAULT_HOMONYMS_PROPERTIES;
+				}
+				ctrl.isMultipleSelect = true;
+				ctrl.asyncPagination = false;
+				ctrl.useCustomFilter = !!attrs.customFilter;
+			}
+		};
+	})
+
+	.controller("luidUserPickerController", ['$scope', '$http', 'moment', '$timeout', '$q', function ($scope, $http, moment, $timeout, $q) {
+		var ctrl = this;
+		var promise; // store the current get request to fetch users
```

Comment:
you should name this promise `getUserPromise` and define it just before the method `getUserAsync`